lib/models/ente_file.dart

```diff
@@ -1,14 +1,12 @@
 // EnteFile is base file entry for various type of files
 // like DeviceFile,RemoteFile or TrashedFile
 
-// @dart=2.9
-
 abstract class EnteFile {
   // returns cacheKey which should be used while caching entry related to
   // this file.
-  String cacheKey();
+  String? cacheKey();
 
   // returns localIdentifier for the file on the host OS.
   // Can be null if the file only exist on remote
-  String localIdentifier();
```

Comment:
@ua741 this isn't being used anywhere as of now. It is being just overridden in file.dart and not used. Should it be removed?

Reply:
yes, can remove it.